twill-core/src/main/java/org/apache/twill/internal/AbstractTwillController.java

```diff
@@ -137,7 +138,7 @@ public final synchronized ServiceDiscovered discoverService(String serviceName)
 
   @Override
   public final ListenableFuture<Set<String>> restartInstances(Map<String,
-      ? extends Set<Integer>> runnableToInstanceIds) {
+    ? extends Set<Integer>> runnableToInstanceIds) {
```

Comment:
I cannot change the type to Future for this method, there is a another method calling it which requires the return type to be a ListenableFuture